10up/wp-newrelic
# Add build zip action

### Description of the Change

Closes https://github.com/10up/wp-newrelic/issues/72

### How to test the Change
- Go to https://github.com/10up/wp-newrelic/actions/runs/10004735251/job/27654012194
- Confirm the Generate ZIP file has  zip that is the plugin files and can be installed / activated.  

### Changelog Entry
N/A

### Checklist:
